.gitignore

```diff
@@ -1,15 +1,18 @@
+*.cache1
+*.cache2
```

Comment:
currently, running "mkosi" without "-f" (nor "-i") means: build if output.raw doesn't exist yet, error out if "output.raw" already exists. running "mkosi" with "-f" (but without "-i") means: build output.raw, possibly removing it first if it already exists.

Now, the new "-i" doesn't change that behaviour: whether -f is specified once or no times just has the effect whether output.raw is removed first or not. However if you now specify it twice then we'll not only remove output.raw but also output.raw.cache1 and output.raw.cache2.

Summary:

 1. mkosi -i → neither delete/replace output.raw nor output.raw.cache[12], use the latter if they exist
 2. mkosi -if → delete/replace output.raw, but not output.raw.cache[12], use the latter if they exist
 3. mkosi -iff → delete/replace output.raw and output.raw.cache[12], don't use the latter

Does that make sense?